pandas/core/arrays/numpy_.py

```diff
@@ -35,52 +35,68 @@ class PandasDtype(ExtensionDtype):
 
     Parameters
     ----------
-    dtype : numpy.dtype
+    dtype : object
+        Object to be converted to a NumPy data type object.
```

Comment:
i see how this is more accurate than just "numpy.dtype", but can we be more specific here?

Reply:
it's the types accepted by np.dtype which is more permissive than just another np.dtype. Object is  straight out of the numpy docs.

do you want to restrict this as before, or establish the actual types accepted by numpy?

Reply:
if its straight out of the numpy docs, then sounds fine